src/pages/Agent/Agent.tsx

```diff
@@ -1,11 +1,110 @@
+import { useParams } from "react-router-dom";
```

Comment:
Crees que puedas crear un archivos css y poner con nombre de las clases estilo [BEM](https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/) ahí las clases de tailwind con el `@apply`?